frontend/package.json

```diff
@@ -102,20 +106,18 @@
     ]
   },
   "scripts": {
-    "gen:theme-typings": "chakra-cli tokens src/theme/index.ts || true",
+    "gen:theme-typings": "chakra-cli tokens src/theme/index.ts --out node_modules/@chakra-ui/styled-system/dist/declarations/src/theming.types.d.ts || true",
     "clean:emotion-types": "rimraf node_modules/@emotion/core/types",
     "postinstall": "patch-package && npm run gen:theme-typings && npm run clean:emotion-types && npm --prefix ../shared install",
-    "start": "env-cmd -f .buildtime-env craco --openssl-legacy-provider start",
-    "build": "npm run build:dd-chunk && cross-env CI=false BUILD_PATH='../dist/frontend' INLINE_RUNTIME_CHUNK=false SKIP_PREFLIGHT_CHECK=true craco build && cd ../ && cp -rf frontend/static/js dist/frontend/static",
-    "build:dd-chunk": "webpack --config webpack.dd.config.js",
-    "test": "cross-env SKIP_PREFLIGHT_CHECK=true craco test --silent",
+    "start": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "test": "cross-env SKIP_PREFLIGHT_CHECK=true vitest",
     "eject": "react-scripts eject",
```

Comment:
no need this anymore

Reply:
2856457124b1c2e7c0d27697e82661f9df658916